lib/sms_autofill.dart

```diff
@@ -184,6 +187,7 @@ class _PhoneFieldHintState extends State<PhoneFieldHint> {
   @override
   void initState() {
     _controller = widget.controller ?? widget.child?.controller ?? _createInternalController();
+    _inputFormatters = widget.inputFormatters ?? widget.child?.controller ?? [];
```

Comment:
Not sure `widget.child?.controller  ` is right here :)

Reply:
ah yes! Will fix sorry!

I have a few changes my side but wanted to introduce them one at a time. Sorry for all the PRs in advance.

Reply:
@jaumard this is fixed 👍

Reply:
Don't be sorry to contribute ! I'm glad you did ;)